docs/general/attribute-naming.md

```diff
@@ -116,6 +116,12 @@ denote old attribute names in rename operations).
   the following Unicode code points: Latin alphabet, Numeric, Underscore, Dot
   (as namespace delimiter).
 
+> Note:
```

Comment:
note to the reviewers:

there is a section above this note saying

> Semantic conventions MUST limit names to printable Basic Latin characters
  (more precisely to
  [U+0021 .. U+007E](<https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)#Table_of_characters>)
  subset of Unicode code points). It is recommended to further limit names to
  the following Unicode code points: Latin alphabet, Numeric, Underscore, Dot
  (as namespace delimiter).

I don't want to change stable spec, and want to emphasize that it's not a semconv limitation, but a tooling one. We can relax tooling limitation in the future up to what's recommended in that section.